src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/AppointmentsService.kt

```diff
@@ -25,6 +25,7 @@ class AppointmentsService(
   fun createAppointmentsForCourt(videoBooking: VideoBooking, prisoner: PrisonerDetails, user: User) {
     checkCourtAppointments(prisoner.appointments, prisoner.prisonCode!!, user)
 
+    // We don't need to do a check for non-prison users as this is covered by the overlapping appointment check.
```

Comment:
Turns out this is covered for other users in the overlapping appointment checking (which is not done for prison users), added comment to further explain why it is the way it is.